packages/neuron-ui/src/components/Settings/Wallets.tsx

```diff
@@ -1,144 +1,134 @@
-  margin-top: 15px;
-  width: 500px;
-  height: 60px;
-  border-radius: 10px;
-  text-align: center;
-  line-height: 60px;
-  background: #ffffff;
-  box-shadow: 1px 1px 3px #999999;
-`
-
-const ContentPanel = styled.div`
-  display: flex;
-  flex-direction: column;
-  justify-content: center;
-  align-items: left;
-  margin: 30px;
-`
-
-const WalletSelectedItem = styled(WalletItem)`
-  background: #81fced;
-`
-
-const WalletButtonPanel = styled.div`
-  display: flex;
-  flex-direction: row;
-`
-
-const WalletButton = styled.div`
-  width: 80px;
-  height: 40px;
-  align-items: left;
-  background: blue;
-  color: white;
-  text-align: center;
-  line-height: 40px;
-  border-radius: 5px;
-`
-
-const AddWallet = styled(WalletButton)`
-  text-transform: capitalize;
-  margin: 30px 0 0 50px;
-`
-
-const DeleteWallet = styled(WalletButton)`
-  text-transform: capitalize;
-  margin: 30px 0 0 50px;
-`
-
-const UpdateWallet = styled(WalletButton)`
-  text-transform: capitalize;
-  margin: 30px 0 0 50px;
+const Popover = styled.div`
+  position: relative;
+  &:hover {
+    & > ul {
+      display: block !important;
+    }
+  }
 `
 
-const wallets: string[] = [
-  'Wallet Name1',
-  'Wallet Name2',
-  'Wallet Name3',
-  'Wallet Name4',
-  'Wallet Name5',
-  'Wallet Name6',
-]
+const WalletActions = ({ isDefault, actionItems }: { isDefault: boolean; actionItems: DropDownItem[] }) => {
+  if (isDefault) {
+    actionItems.splice(0, 1)
+  }
+  return (
+    <Popover>
+      <Configure />
+      <Dropdown
+        items={actionItems}
+        style={{
+          position: 'absolute',
+          top: '100%',
+          right: '0',
+          zIndex: '999',
+          display: 'none',
+        }}
+      />
+    </Popover>
+  )
+}
 
-const General = () => {
+const Wallets = (props: React.PropsWithoutRef<ContentProps & RouteComponentProps>) => {
+  const { wallets } = useContext(WalletContext)
   const [t] = useTranslation()
-  const [walletSelected, setWalletSelected] = useState(wallets[0])
-  const [walletsState, setWalletsState] = useState(wallets)
 
-  const deleteWallet = () => {
-    const temps = walletsState
-    const temp = walletSelected
-    const index = temps.indexOf(temp)
-    if (index > -1) {
-      temps.splice(index, 1)
-    }
-    setWalletSelected(temps[0])
-    setWalletsState(temps)
-  }
+  const [walletSelected, setWalletSelected] = useState(() => {
+    getWallets()
+    return 0
+  })
 
-  const addWallet = () => {
-    const temps = walletsState
-    setWalletsState(temps.concat(`Wallet Name${temps.length + 1}`))
-  }
-
-  const updateWallet = () => {
-    const temps = walletsState
-    const temp = walletSelected
-    const index = temps.indexOf(walletSelected)
-    const name = prompt(t('Please enter your wallet name'), temp)
-    if (index > -1 && name != null && name !== '') {
-      temps.splice(index, 1, name)
-    }
-    setWalletsState(temps)
-  }
+  const actionItems = (index: number) => [
+    {
+      label: t('select'),
```

Comment:
@BaojunCZ @Keith-CY @duanyytop Please use `feature.section.select` style of `i18n` keys. `feature.section` could be anything that helps us distinguish where we should organize that translation. Using a single word as key would make very hard to translate and organize, and cause errors when we update. (e.g., when we want to call `Edit` on manage wallet view as `Update`, and keep as `Edit` on another view. Just an example.)

Reply:
ok